vcd/datasource_vcd_catalog.go

```diff
@@ -124,20 +153,26 @@ func datasourceVcdCatalogRead(ctx context.Context, d *schema.ResourceData, meta
 		return diag.Errorf("There was an issue when retrieving metadata - %s", err)
 	}
 
-	dSet(d, "description", catalog.Catalog.Description)
-	dSet(d, "created", catalog.Catalog.DateCreated)
-	dSet(d, "name", catalog.Catalog.Name)
-	d.SetId(catalog.Catalog.ID)
-	if catalog.Catalog.PublishExternalCatalogParams != nil {
-		dSet(d, "publish_enabled", catalog.Catalog.PublishExternalCatalogParams.IsPublishedExternally)
-		dSet(d, "cache_enabled", catalog.Catalog.PublishExternalCatalogParams.IsCachedEnabled)
-		dSet(d, "preserve_identity_information", catalog.Catalog.PublishExternalCatalogParams.PreserveIdentityInfoFlag)
+	dSet(d, "description", catalog.AdminCatalog.Description)
+	dSet(d, "created", catalog.AdminCatalog.DateCreated)
+	dSet(d, "name", catalog.AdminCatalog.Name)
+
+	d.SetId(catalog.AdminCatalog.ID)
+	if catalog.AdminCatalog.PublishExternalCatalogParams != nil {
+		dSet(d, "publish_enabled", catalog.AdminCatalog.PublishExternalCatalogParams.IsPublishedExternally)
+		dSet(d, "cache_enabled", catalog.AdminCatalog.PublishExternalCatalogParams.IsCachedEnabled)
+		dSet(d, "preserve_identity_information", catalog.AdminCatalog.PublishExternalCatalogParams.PreserveIdentityInfoFlag)
 	}
 
 	err = d.Set("metadata", getMetadataStruct(metadata.MetadataEntry))
 	if err != nil {
 		return diag.Errorf("There was an issue when setting metadata into the schema - %s", err)
 	}
 
+	err = setCatalogRecordValuesToSchema(d, adminOrg, catalog.AdminCatalog.Name)
```

Comment:
usually we call this like this: setCatalogData

Reply:
Done ✅